packer/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+pushd `dirname "$0"`
+
+if [ -e manifest.json ]; then
```

Comment:
This is something you would manually do as part of a pull request that requires a new image. We stole the pattern from https://github.com/PillarTechnology/PillarWebReverseProxy